docs/user/code_structure.rst

```diff
@@ -736,6 +736,65 @@ restrictions on the number of MooringMatrix blocks.
    :width: 400 pt
    :align: center
 
+.. _user-code-structure-cable-class:
+
+PTO-Sim Class
+^^^^^^^^^^^^^^^^
+
+The PTO-Sim class contains all the information for the PTO-Sim blocks, which can be used to simulate PTO systems. 
+The difference beetween the PTO-Sim class and the PTO class is that the PTO-Sim class have detailed models of different components
+that are used in PTO systems such as hydraulic cylinders, hydraulic accumulators, hydraulic motors, electric generators, etc., 
+while the PTO class have a linear parametric model that summarizes the PTO dynamics with a stiffness and a damping term.
+At a high level, the PTO-Sim class interacts with the rest of 
+WEC-Sim as shown in the diagram below:
+
+.. figure:: /_static/images/code_structure/PTOSimClass_diagram.png
+   :width: 750pt
+
+The PTO-Sim blocks receive the linear or angular response from the PTO blocks and give either the torque or the force depending on the PTO dynamics.
+
+PTO-Sim Class Initialization
+""""""""""""""""""""""""""""""""
+The properties of the PTO-Sim class (``PTOSimClass``) are defined in the ``PTOSimBlock`` object. The PTO-Sim class must be
+initialized in the ``wecSimInputFile.m`` script. There are three properties that must be initialized for all the PTO-Sim blocks,
+those are the name, the block number, and the type:: 
+
+    PTOSimBlock(i) = cableClass('PTOsimName');
```

Comment:
@jleonqu I believe `cableClass` should be `ptoSimClass`, but can you confirm and update as required.